lib/go-tc/deliveryservices.go

```diff
@@ -226,11 +225,8 @@ type DeliveryServiceNullableV12 struct {
 // for all fields to be null.
 // TODO move contents to DeliveryServiceNullableV12, fix references, and remove
 type DeliveryServiceNullableV11 struct {
-	// NOTE: the db: struct tags are used for testing to map to their equivalent database column (if there is one)
-	//
 	Active                   *bool                   `json:"active" db:"active"`
 	AnonymousBlockingEnabled *bool                   `json:"anonymousBlockingEnabled" db:"anonymous_blocking_enabled"`
-	CacheURL                 *string                 `json:"cacheurl" db:"cacheurl"`
```

Comment:
It looks like you re-added a commit where you took `CacheURL` out of V11 DS structs - there was a previously resolved comment about this so I think it was possibly a rebase issue?

Reply:
It was intentional, I readded it for backwards compatibility with older TC servers, but the `json` serializer handles it for us such that the presence or absence of that field does not break anything.

Reply:
It doesn't break anything as far as JSON serialization, but it is a breaking change to old client call signatures. Changes to old client call signatures must be non-breaking.